db/migrate/20151016094008_create_forums.rb

```diff
@@ -0,0 +1,39 @@
+class CreateForums < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [24/15]<br>Method has too many lines. [28/10]